.github/workflows/changelog.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: "Check for change log fragments"
+on:
+  pull_request:
+    branches:
+      - "develop"
+      - "next"
+
+jobs:
+  changelog:
```

Comment:
Better suited to be added to `ci_pullrequest.yml`?

Reply:
I must have had a good reason for putting it in a separate file but I can't remember what it was. I went ahead and moved it over.